softwareComponents/atoms/include/atoms/util.hpp

```diff
@@ -32,3 +32,31 @@ std::make_unsigned_t< T > to_unsigned( T value ) {
     }
     return static_cast< std::make_unsigned_t< T > >( value );
 }
+
+template < typename Container, typename T >
+bool contains( const Container& c, const T& value ) {
+    for ( const auto& x : c )
+        if ( x == value )
+            return true;
+    return false;
+}
+
+/**
+ * Given an iterable list of items, return index of it. If element is not found,
+ * -1 is returned.
+ */
+template < typename T, typename Ts >
+int indexOf( T t, const Ts& source ) {
```

Comment:
Is it really needed? It can be useful to find an index even I cannot index the sequence. `static_assert` could also yield false positives (e.g., for `std::map< int, int >`).